.flowconfig

```diff
@@ -39,6 +39,9 @@
 ; Ignore react-native-mock
 .*/node_modules/react-native-mock/*
 
+; Ignore the example app
```

Comment:
We shouldn't ignore the example app. Using Flow in examples (and tests) makes sure Flow is not broken for our public API.